### DISCUSSION TITLE
Imported project incorrectly assigned MIT License

### DISCUSSION THREAD

**Discussion by scotmcp** [community/community#132801] [Repositories] [answered]
### Select Topic Area

Question

### Body

How do I change the license listed on a repo. I have a private repo that I imported from another platform, and the repo got labelled with the MIT License. This repo is proprietary and not publically licensed at all. How do I fix this?

**Comment by knarrff** [comment, marked as answer]
Hi @scotmcp

Does your repo contain a LICENSE, LICENSE.txt or LICENSE.md file or similar? If so, remove it. Also, look out for SPDX records within the files. github parses for them and uses the license mentioned there. If there are any, also change/remove those. This should in theory be sufficient to let github choose another license. Without further information (especially without the content of the repo) it is hard to say more.

**Comment by scotmcp** [comment]
Thanks, that did the trick....I did not create that file, and I don't know how it got there.  I imported 4 other projects to github at the same time and they did not have the same problem.

Appreciate the help

**Reply by knarrff** [reply]
Hi @scotmcp 

I am glad this worked. Maybe could I ask a small favor: it might help others if you could mark the question as answered as well.

Thanks

**Reply by scotmcp** [reply]
done thanks
